src/sentry/static/sentry/app/views/performance/transactionDetails/eventMetas.tsx

```diff
@@ -83,15 +98,16 @@ function EventMetas({event, organization, projectId, location, quickTrace}: Prop
   );
 }
 
-const EventDetailHeader = styled('div')`
+const EventDetailHeader = styled('div')<{type?: 'transaction' | 'event'}>`
   display: grid;
-  grid-template-columns: repeat(3, 1fr);
+  grid-template-columns: repeat(${p => (p.type === 'transaction' ? 3 : 2)}, 1fr);
   grid-template-rows: repeat(2, auto);
   grid-gap: ${space(2)};
   margin-bottom: ${space(2)};
 
   @media (min-width: ${p => p.theme.breakpoints[1]}) {
-    grid-template-columns: minmax(160px, 1fr) minmax(160px, 1fr) minmax(160px, 1fr) 6fr;
+    grid-template-columns: minmax(160px, 1fr) ${p =>
+        p.type === 'transaction' ? 'minmax(160px, 1fr)' : ''} minmax(160px, 1fr) 6fr;
```

Comment:
I know this is going to be more LOC, but can we do something like
```javascript
${p => p.type === 'transaction' ? 'grid-template-columns: ...' : 'grid_template-columns: ...'}
```
this is probably a lot more legible.